frontend/views/components/Chatroom/ConversationsList.vue

```diff
@@ -1,6 +1,6 @@
 <template lang="pug">
 .c-conversation-list
-  label.label {{title}}
+  i18n.label(tag='label') {{title}}
```

Comment:
One note on how `i18n` and `L` work: they only accept literal strings, so since `{{title}}` is a variable, you do not need to use `i18n` here, but rather wherever `title` is originally defined as a string literal, that is where you would use the `L` function

Reply:
Good point